### PR TITLE
Misc. changes

### PR DESCRIPTION
**Pull request by saurabhagas**
See commit list.

Have a few more changes planned for this PR. Feel free to merge it if the changes are okay and builds are green, however.

### CODE REVIEW

File: dynamic-config/model/src/main/java/org/terracotta/dynamic_config/api/model/Requirement.java [hunk 1]
```diff
@@ -31,9 +31,9 @@
   NODE_RESTART,
 
   /**
-   * Setting change needs only active servers to be online plus eventually some passive servers, but not all
+   * Setting change needs only active servers to be online plus the targeted objects which can be nodes, stripes or the cluster
    */
-  ACTIVES_ONLINE,
+  TARGET_ONLINE,
```
Comment (mathieucarbou) [inline dynamic-config/model/src/main/java/org/terracotta/dynamic_config/api/model/Requirement.java:36]: With an -S- ?

**Review by mathieucarbou — APPROVED** [review]

**Comment by mathieucarbou** [comment]
Some tests need to be updated

File: dynamic-config/model/src/main/java/org/terracotta/dynamic_config/api/model/Requirement.java (hunk 1 above)
Comment (saurabhagas) [reply dynamic-config/model/src/main/java/org/terracotta/dynamic_config/api/model/Requirement.java]: Done

**Comment by saurabhagas** [comment]
> Some tests need to be updated

Done

**Pull request merged**